Makefile

```diff
@@ -60,6 +60,11 @@ clean: #@ Clean the test and build cache and remove binaries
 test:| go_version_check #@ Run the tests
 	@go test -cover $(PACKAGES)
 
+fuzz:| go_version_check #@ Run the fuzz tests
```

Comment:
Question: How long do they take? Should we consider them for any pipeline?

Reply:
If no argument is passed, the test will run forever or until an error is found. While the test run the fuzz engine will keep generating new inputs. At the moment I've set 1' timeout but we can change it. I think it makes sense to have a very long test run (previous PR was considering 30' each test) and not run these kind of tests in the pipeline every PR/merge action.

Reply:
Perfect, thanks for the explanation